### PR TITLE
releaseCheck.sh: use API query and update counts for SBOM .sig file

### PR DESCRIPTION
**Pull request by sxa**
Release tool will not be able to do its checks in the job output without this

Signed-off-by: Stewart X Addison <sxa@redhat.com>

### CODE REVIEW

File: sbin/releaseCheck.sh [hunk 1]
```diff
@@ -10,122 +10,126 @@ TEMURIN_TAG=$2
...
 then
    echo "Source images: OK!"
 else
    echo "Source images: Not complete:"
-   [ ! -z "$VERBOSE" ] && cat releaseCheck.$$.tmp | grep sources | grep href | cut -d'"' -f2
+   [ ! -z "$VERBOSE" ] && cat releaseCheck.$$.tmp | grep sources
 fi
 
-rm releaseCheck.$$.tmp
+# rm releaseCheck.$$.tmp
```
Comment (karianna) [inline sbin/releaseCheck.sh:135]: Why commented out?

File: sbin/releaseCheck.sh [hunk 2]
```diff
@@ -10,122 +10,126 @@ TEMURIN_TAG=$2
...
+  ACTUAL=$(cat releaseCheck.$$.tmp | grep ${ARCH}_windows | wc -l)
   if [ $ACTUAL -eq $EXPECTED ]
   then
      echo "Windows on $ARCH: OK!"
   else
     if [ $ACTUAL -eq 0 ]; then
       echo "Windows on $ARCH: Not published"
     else
-      echo "Windows on $ARCH: INCOMPLETE: $EXPECTED (Expect jre, jdk, msi-jre msi-jdk, debugimage, static-libs (Not JDK8) in base, json, sha256, GPG sig, plus 2 SBOMs"
+      echo "Windows on $ARCH: INCOMPLETE: $ACTUAL/$EXPECTED  (Expect jre, jdk, msi-jre msi-jdk, debugimage, static-libs (Not JDK8) in base, json, sha256, GPG sig, plus 3 SBOMs"
```
Comment (karianna) [inline sbin/releaseCheck.sh:101]:
```suggestion
      echo "Windows on $ARCH: INCOMPLETE: $ACTUAL/$EXPECTED (Expect jre, jdk, msi-jre msi-jdk, debugimage, static-libs (Not JDK8) in base, json, sha256, GPG sig, plus 3 SBOMs"
```

**Review by karianna — CHANGES_REQUESTED** [review]

File: sbin/releaseCheck.sh (hunk 2 above)
Comment (sxa) [reply sbin/releaseCheck.sh]: Sorted

**Review by karianna — APPROVED** [review]
LGTM

**Review by karianna — APPROVED** [review]
LGTM

**Review by gdams — APPROVED** [review]

**Pull request merged**